docs/DeveloperGuide.md

```diff
@@ -989,6 +997,61 @@ testers are expected to do more *exploratory* testing.
 
 2. _{ more test cases …​ }_
 
+### Editing a person
+
+1. Test case 1: Editing an existing student's own detail 
+   1. Prerequisites
+      2. Add student: `add n/John p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/Chemistry t/Sec 3 nok/ n/Jack Doe p/10987654 e/jackd@example.com a/311, Clementi Ave 2, #02-25`
+      3. Note the student and his or her index in the displayed list in the `Student` tab.
+   2. Edit the student (assuming the student's index is 1): `edit 1 n/updated student p/99993293 e/edited@gmail.com a/edited address t/`
+   3. Expected: All fields are changed according to `edit` command inputs and all tags are removed.
+2. Test case 2: Edit an existing student's NOK detail
+   1. Prerequisites: same as above
```

Comment:
Capital letter for same? ie 'Same' and the rest as well

Reply:
good catch!